WikilogUtils.php

```diff
@@ -210,7 +210,16 @@ public static function parsedArticle( Title $title, $feed = false ) {
 		$parser->startExternalParse( $title, $parserOpt, Parser::OT_HTML );
 
 		# Parse article.
-		$arttext = $article->getText();
+		$articleContent = $article->getContent();
+		if ( !($articleContent instanceof TextContent) ){
+			# Restore default behavior.
```

Comment:
From line 230, I think we need to restore the default behavior (I think it's related to the parser).
```
	/**
	 * Enable special wikilog feed parsing.
	 *
	 * This function changes the parser behavior in order to output
	 *
	 * The proper way to use this function is:
	 * @code
	 *   $saveFeedParse = WikilogParser::enableFeedParsing();
	 *   # ...code that uses $wgParser in order to parse articles...
	 *   WikilogParser::enableFeedParsing( $saveFeedParse );
	 * @endcode
	 *
	 * @note Using this function changes the behavior of Parser. When enabled,
	 *   parsed content should be cached under a different key.
	 */
	public static function enableFeedParsing( $enable = true ) {
		$prev = self::$feedParsing;
		self::$feedParsing = $enable;
		return $prev;
	}
```